subfun/design/setSpeedTargets.m

```diff
@@ -17,7 +17,7 @@
         % Outputs an "empty" matrix in case no speed task is required
     else
         [~, nbEventsPerBlock, ~, nbBlocks] = getDesignInput(cfg);
-        speeds = ones(nbBlocks, nbEventsPerBlock) * cfg.dot.speedPixPerFrame;
+        speeds = ones(nbBlocks, nbEventsPerBlock);
```

Comment:
I changed this:
- "fixation cross" should not have speed
- also without this change both "sides" of the `if / else` block had the same content making it effectively useless

```matlab
if A
  b = 2
else
  b = 2
end
```

Reply:
and that multiplication is actually important so will put it back

Reply:
ah OK then. Sorry about that. 
let's remove the IF block for now because this is confusing and add a `TODO` explaining that things need to be change there later.

Reply:
Done